lib/launcher.js

```diff
@@ -351,11 +351,16 @@ class Launcher {
 
     /**
      * emit event from child process to reporter
-     * @param  {Object} m  event object
+     * @param  {String} cid
+     * @param  {Object} m event object
      */
-    messageHandler (m) {
+    messageHandler (cid, m) {
```

Comment:
@KOPTE3 usually yes but the changes are so small that I would be fine to have it in this PR but happy to merge 2 PRs as well 😉